ushahidi/platform-mobile
# Fix/geocoding and edit

This pr:
- Allows editing of pending posts
- Always allow searching for an address
- Fixes to geocoding functionality


Test this by:
- Turn off network/wifi or location
- Add a post with locations, Type something in the "Search for address..." form
- Submit post
- Go to "View responses"
- Click on the pen-icon on the just added, pending post
- Turn on the network
- Search for an address you know exist
- [ ] It should show that address on the map + add the lat/long
- Click on the "Use my location"
-  [ ] It should show that location on the map + add the lat/long
- Save the post
- [ ] It should submit the post

fixes ushahidi/platform#4220
